charts/scrutiny/charts/collector/templates/daemonset.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "collector.fullname" . }}
+  labels:
+    {{- include "collector.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "collector.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "collector.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "collector.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          envFrom:
+            - secretRef:
+                name: {{ include "collector.fullname" . }}-credentials
+          env:
+            {{- range $key, $value := .Values.env }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - mountPath: /run/udev
+              name: udev
+      volumes:
+        - name: udev
```

Comment:
on K8s, the collector will probably need to run as `privileged` or mount the `/dev/` directory (which is basically the same thing). 

https://github.com/AnalogJ/scrutiny/issues/229

basically we need to make sure that the collector container has access to all the storage devices. When running docker manually, my README instructions state that users should pass each device individually using `--device` but that won't be possible in K8s.

Reply:
Right now the collector runs with `privileged: true` and the web does not, so that should be ok. I am not sure about whether the `web` can run as a non-root user though.

`privileged: true` already mounts `/dev`, does `/run/udev` need to be mounted as well or can I get rid of it?

Reply:
From your comment [here](https://github.com/AnalogJ/scrutiny/pull/363#issuecomment-1236157305) I understand there's nothing to change then :)

Reply:
yeah, sorry I must have skimmed over the `privileged` config